test/SourceKit/CodeComplete/complete_generic_property_wrapper_initialization.swift

```diff
@@ -0,0 +1,32 @@
+// RUN: %complete-test %s -tok=COMPLETE_GENERIC
```

Comment:
`complete-test` is SourceKit test. For non-SourceKit specific changes we use `swift-ide-test` in `test/IDE/complete_*.swift`. But in this specific case, please use `validation-test/IDE/crashers_2_fixed/`.

Reply:
Ah, didn’t know about `validation-test/IDE/crashers_2_fixed`. Moved it there.